README.md

```diff
@@ -100,13 +110,15 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema details the basic structure of the tables in the database. Essentially, it describes each table's columns and what their data type is as well as sets the relationships the tables have with eachother.
+
+A one-to-many relationship would be shown using "belongs_to" and "has_many".
 ```
```

Comment:
True, we would use `belongs_to` and `has_many` to indicate associations using ActiveRecord. But in our schema, we would indicate the same thing using a foreign key (e.g., `instructor_id`)